hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -415,8 +416,8 @@ default R scan(String table, Set<byte[]> prefixes) {
         /**
          * Scan records by rowkey start and prefix from a table
          */
-        default R scan(String table, byte[] startRow,
-                              boolean inclusiveStart, byte[] prefix) {
+        default R scan(String table, byte[] startRow, boolean inclusiveStart,
```

Comment:
prefer to wrap line before "boolean inclusiveStart"

Reply:
this line only 78 char.